03-client-api/references/options.yml

```diff
@@ -10,11 +10,11 @@ description:
     - `infer`: whether to enable inference for the provided query (only settable at transaction level and above) (default: `false`)
```

Comment:
`prefetchSize` was still going by its old name `batchSize`, and there was a typo in `schemaLockAcquireTimeoutMillis` - both are now fixed.